resolwe_bio/processes/chip_seq/prepeak_qc.yml

```diff
@@ -0,0 +1,249 @@
+# =====================
+# ChIP-Seq - Prepeak QC
+# =====================
+---
+
+- slug: qc-prepeak
+  name: Pre-peakcall QC
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/chipseq:3.0.0
+    resources:
+      memory: 16384
+      cores: 10
+  data_name: "{{ alignment.bam.file|basename|default('?') }}"
+  version: 0.0.1
+  type: data:prepeakqc
+  category: ChIP-Seq:QC report
+  flow_collection: sample
+  persistence: CACHED
+  description: >
```

Comment:
Please add more extensive description, explaining also the output format and results field names. You can use hyperlinks to the web resources explaining key metrics results, e.g. NSC, RSC.